src/screens/Library.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+
+import Card from "../components/Card";
+import Loader from "../components/Loader";
+import Button from "../components/Button"
+
+import { getData } from "../services/Services";
+
+const Library = (props) => {
+    const [cardComponent, setCardComponent] = useState(null)
+    const [user, setUser] = useState("candidate")
+
+    useEffect(() => {
+        const type = user === "company" ? "candidates" : "labs";
+        getData(type).then(data => {
+            setCardComponent(
+                <>
+                    <Button
+                        size="large"
+                        variant="primary"
+                        onClick={() => { user === "company" ? setUser("candidate") : setUser("company") }}
+                    >
+                        {"Switch to: " + user}
```

Comment:
Won't this write _Switch to company_ when you're already a company?

Reply:
Yes, that's true. Initially I made it a simple button with the value of user as its child, then added the 'Switch to: ' part and I wasn't paying attention. Thanks :)